tests/unit/browser/frontend/lambda/code/test_cloudfront.py

```diff
@@ -0,0 +1,32 @@
+from unittest import mock, TestCase
+
+from browser.frontend.lambdas.code.cloudfront import get_cloudfront_distribution, invalidate_distributions
+
+
+class TestCloudfront(TestCase):
+    @mock.patch("browser.frontend.lambdas.code.cloudfront.cloudfront_client.list_distributions")
+    def test_get_cloudfront_distribution(self, mock_list_distributions):
+        mock_list_distributions.return_value = {
+            "DistributionList": {
+                "Items": [
+                    {"Id": "id1", "Origins": {"Quantity": 1, "Items": [{"DomainName": "test_bucket-one-match"}]}},
+                    {"Id": "id2", "Origins": {"Quantity": 1, "Items": [{"DomainName": "test_bucket"}]}},
+                ]
+            }
+        }
+        with self.subTest("Single distribution"):
+            self.assertListEqual(get_cloudfront_distribution("test_bucket-one-match"), ["id1"])
+        with self.subTest("Multiple distributions"):
+            self.assertListEqual(get_cloudfront_distribution("test_bucket"), ["id1", "id2"])
+        with self.subTest("No Match"):
+            self.assertListEqual(get_cloudfront_distribution("no-match"), [])
+
+    @mock.patch("browser.frontend.lambdas.code.cloudfront.cloudfront_client" ".create_invalidation")
```

Comment:
Does this work?

Reply:
it does work, but that was "black" messing up my formatting. I fixed it.